cobbler.spec

```diff
@@ -1,19 +1,22 @@
-# Copyright (c) 2006 Michael DeHaan <mdehaan@redhat.com>
 #
-# RPM spec file for all Cobbler packages
+# spec file for package cobbler
 #
-# Supported/tested build targets:
-# - Fedora: 30, 31, Rawhide
-# - CentOS + EPEL: 7, 8
-# - SLE: 15sp1
-# - openSUSE: Leap 15.1, Tumbleweed
-# - Debian: 10
-# - Ubuntu: 18.04
+# Copyright (c) 2021 SUSE LLC
+# Copyright (c) 2006 Michael DeHaan <mdehaan@redhat.com>
 #
-# If it doesn't build on the Open Build Service (OBS) it's a bug.
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-# Force bash instead of Debian dash
+
```

Comment:
```suggestion
# Force bash instead of Debian dash
```

For me this comment is useful and I wouldn't know why that is here if the comment was missing.